gma/src/common/native_ad_internal.cc

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "gma/src/common/native_ad_internal.h"
+
+#include "app/src/include/firebase/future.h"
+#include "app/src/include/firebase/internal/mutex.h"
+#include "app/src/include/firebase/internal/platform.h"
+#include "app/src/reference_counted_future_impl.h"
+#include "gma/src/include/firebase/gma/internal/native_ad.h"
+
+#if FIREBASE_PLATFORM_ANDROID
+#include "gma/src/android/native_ad_internal_android.h"
+#else
+#include "gma/src/stub/native_ad_internal_stub.h"
+#endif  // FIREBASE_PLATFORM_ANDROID, FIREBASE_PLATFORM_IOS,
+        // FIREBASE_PLATFORM_TVOS
```

Comment:
Why do we list TVOS?

Reply:
That's right. Removed the residual comment. Will add it back when we add support for iOS and tvOS.